### PR TITLE
Change unit to ANMO

### LINKED ISSUE #2: Change the ticker to ANMO
By default it is Unit, change it to ANMO

### PR DESCRIPTION
**Pull request by aelesbao**
Closes #2

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -38,6 +38,14 @@ pub fn authority_keys_from_seed(s: &str) -> (AuraId, GrandpaId) {
 	)
 }
 
+pub fn chain_properties() -> Properties {
+	let mut properties = Properties::new();
+	properties.insert("tokenDecimals".into(), 9.into());
+	properties.insert("tokenSymbol".into(), "ANMO".into());
+	properties.insert("ss58Format".into(), 64.into());
```
Comment (nblogist) [inline node/src/chain_spec.rs:45]: @aelesbao is another network having the same address format?

**Review by nblogist — COMMENTED** [review]
Please look into the address format, else we can go with `14` if none other network is using it and  64 is being used.

**Comment by nblogist** [comment]
Seems like it is available 

 ```
HydraDXAccount =>
         (63, "hydradx", "HydraDX standard account (*25519).")
     AventusAccount =>
         (65, "aventus", "Aventus Chain mainnet, standard account (*25519).")
```
64 is available

Thanks @aelesbao

**Review by nblogist — APPROVED** [review]
Looks good!

**Comment by aelesbao** [comment]
> ```
> HydraDXAccount =>
>         (63, "hydradx", "HydraDX standard account (*25519).")
>     AventusAccount =>
>         (65, "aventus", "Aventus Chain mainnet, standard account (*25519).")
> ```
> 
> 64 is available

Just as a reference for where we found it: Substrate has a full list of the [ss58 address format in use](https://github.com/paritytech/substrate/blob/master/primitives/core/src/crypto.rs#L479).

**Pull request merged**